ansible/playbooks/roles/common/handlers/main.yml

```diff
@@ -10,3 +10,7 @@
     daemon_reload: true
     name: sshd
     state: reloaded
+
+- name: reload sysctl
+  become: true
```

Comment:
is this needed? we usually run as root, and not sure how well sudo works...

Reply:
Yea, I guess we can skip it.